bump_version.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# bump_version.sh (show|major|minor|patch|prerelease|build)
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+VERSION_FILE=src/example/_version.py
+
+HELP_INFORMATION="bump_version.sh (show|major|minor|patch|prerelease|build|finalize)"
+
+old_version=$(sed -n "s/^__version__ = \"\(.*\)\"$/\1/p" $VERSION_FILE)
+
+if [ $# -ne 1 ]
+then
+   echo "$HELP_INFORMATION"
+else
+    case $1 in
+        major|minor|patch|prerelease|build)
+            new_version=$(python -c "import semver; print(semver.bump_$1('$old_version'))")
+            echo Changing version from "$old_version" to "$new_version"
+            tmp_file=/tmp/version.$$
+            sed "s/$old_version/$new_version/" $VERSION_FILE > $tmp_file
+            mv $tmp_file $VERSION_FILE
```

Comment:
Since it may come up in the future and so we don't forget our past, we should probably add a brief comment here about why we do this business with the temp file (short version: Mac OS uses BSD `sed` - see https://github.com/cisagov/cyhy-mailer/commit/283dcbfa893009ec705883d78fc1ce2d64db0a15)

Reply:
That's a good point. I've added comments in 03278ca to explain.